shared/styles/shared.tsx

```diff
@@ -17,16 +17,25 @@ export const globalMargins = {
 }
 /* eslint-enable sort-keys */
 
-export const backgroundModeToColor = {
-  Announcements: globalColors.blue,
-  Documentation: globalColors.blueDarker,
-  HighRisk: globalColors.red,
-  Information: globalColors.yellow,
-  Normal: globalColors.white,
-  Success: globalColors.green,
-  Terminal: globalColors.blueDarker2,
+const _backgroundModeToColor = {
+  Announcements: 'blue',
+  Documentation: 'blueDarker',
+  HighRisk: 'red',
+  Information: 'yellow',
+  Normal: 'white',
+  Success: 'green',
+  Terminal: 'blueDarker2',
 }
 
+export const backgroundModeToColor = new Proxy(
```

Comment:
lets not use a proxy. why not just make backgroundModeToColor a closure and keep it simple

Reply:
or just make those getters instead in that const